packages/server-wallet/src/wallet/index.ts

```diff
@@ -240,6 +244,11 @@ export class Wallet implements WalletInterface {
   onNotification(_cb: (notice: StateChannelsNotification) => void): {unsubscribe: () => void} {
     throw 'Unimplemented';
   }
+
+  // Should be called after wallet creation
```

Comment:
At first, I didn't understand this comment. This is a method defined on an instance of a wallet, so it can only be called after the wallet is created.

Then I thought it means "this _must_ be called after wallet creation". If so, why isn't it just passed to the wallet's constructor?

Reply:
We could pass it into the wallets constructor, there's just nothing else in there right now (though we should also make the environment config injectable see #2502 )

Reply:
Wallet and chain service interaction improvements should be done in v1, resolving comment.